extra/hhvm.conf

```diff
@@ -19,7 +19,7 @@ hhvm.log.header = true
 hhvm.log.access[default][file] = /var/log/hhvm/access.log
 hhvm.log.access[default][format] = "%h %l %u %t \"%r\" %>s %b"
 hhvm.server.source_root = CTFPATH
-hhvm.repo.central.path = /var/cache/hhvm/hhvm.hhbc
+hhvm.repo.central.path = /var/run/hhvm/hhvm.hhbc
```

Comment:
But you call `sudo chown www-data:www-data "$__repofile"` regardless of which one we're using. It seems like more trouble to allow for both with no benefit.